addons/jest/README.md

```diff
@@ -0,0 +1,130 @@
+# Storybook addon Jest
+
+Brings Jest results in storybook.
+
+[![Storybook Jest Addon Demo](@storybook/addon-jest.gif)](https://storybook-addon-jest-example.herokuapp.com/)
+
+> Checkout the above [Live Storybook](https://storybook-addon-jest-example.herokuapp.com/).
+
+## Getting started
+
+### Install
+
+`npm install --save-dev @storybook/addon-jest`
+
+or
+
+`yarn add --dev @storybook/addon-jest`
+
+### Jest Configuration
+
+When running **Jest**, be sure to save the results in a json file:
+
+`package.json`
+
+```js
+"scripts": {
+  "test": "jest --json --outputFile=.jest-test-results.json"
+}
+```
+
+Add it the result file to `.gitignore`:
```

Comment:
@renaudtertrais @ndelangen why do we even have this recommendation? Checking it into VCS makes a lot of sense to me

Reply:
Well, it will likely have tons of merge-conflicts over time?

Reply:
Just like lockfiles and test snapshots. It's easy to regenerate them, so this shouldn't be a problem

Reply:
```js
true
```

I will change the text and make it so users are aware of the trade-offs.